packages/payment-detection/src/eth/fee-proxy-detector.ts

```diff
@@ -61,14 +62,24 @@ export class EthFeeProxyPaymentDetector extends FeeReferenceBasedDetector<Paymen
       throw Error('ETH fee proxy contract not found');
     }
 
-    const proxyInfoRetriever = new EthProxyInfoRetriever(
-      paymentReference,
-      proxyContractArtifact.address,
-      proxyContractArtifact.creationBlockNumber,
-      address,
-      eventName,
-      network,
-    );
+    const proxyInfoRetriever =
```

Comment:
shouldn't there also be a similar thing for any-to-eth?

Reply:
In all places where there could be a distinction between theGraph or web3 info retrievers we are using this [networkSupportsTheGraph](https://github.com/RequestNetwork/requestNetwork/blob/2a884beb1447b7edc02328670275dac1ca1a8563/packages/payment-detection/src/thegraph/index.ts#L33) function.
But in [any-to-eth](https://github.com/RequestNetwork/requestNetwork/blob/eth-payments-thegraph/packages/payment-detection/src/any/any-to-eth-proxy.ts) that same function is not used, so I'm confused how it should be handled there. 
Perhaps @yomarion could clarify this stuff for us.